hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/api/ClusterResource.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.mesos.hdfs.api;
+
+import org.apache.mesos.hdfs.config.HdfsFrameworkConfig;
+import org.apache.mesos.hdfs.state.LiveState;
+import org.apache.mesos.hdfs.state.AcquisitionPhase;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import com.google.inject.Inject;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+/**
+ * Cluster management API.
+ */
+@Path("cluster")
+public class ClusterResource {
+  private final Log log = LogFactory.getLog(ClusterResource.class);
+
+  private HdfsFrameworkConfig config;
```

Comment:
Well, I'm not sure if we'd have to copy the mesos-site.xml file to the executors, since I don't think it actually gets used there (correct me if I'm wrong). However, the mesos-site.xml used by the scheduler would indeed need to be modified with the new value to ensure things go smoothly in a scheduler failover event.

I'd suggest we hold off on config-updating logic until we have a game plan for the configuration as a whole (with the understanding that the config might need to be adjusted if a scaled-up scheduler dies and needs restarting). Open to input, though.